pkg/pack.sh

```diff
@@ -48,7 +48,7 @@ else
     init_distro_name
 fi
 
-__common_parameters="/p:$__targets_param /p:DistroRid=$__distro_rid /verbosity:minimal"
+__common_parameters="/p:$__targets_param /p:DistroRid=$__distro_rid /verbosity:minimal /p:Platform=x64"
```

Comment:
Got it. Pack.sh imports an msbuild props file written by the build. That props file was using the the wrong variable for its <Platform> tag. I pushed a fix.